.githooks/pre-push

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+remote="$1"
+url="$2"
+
+z40=0000000000000000000000000000000000000000
+
+while read local_ref local_sha remote_ref remote_sha
+do
+    echo "Performing linter check"
+    python tools/lint.py . --nproc=5 || exit 1
```

Comment:
Can we run the lint only on the changed files as well?

Reply:
If we move it to a pre-commit hook yes, this is a pre-push hook, so it'd be harder to find all the files that were changed. It's possible but more involved. This takes around 5s, which I find it tolerable, as we normally don't push that often. I've been using it locally for few weeks now and I normally don't notice that is there, except when I break the lint :)

Reply:
Done